### PR TITLE
Update Croatian transltion

### PR DESCRIPTION
**Pull request by milotype**
Just a minor typo correction

### CODE REVIEW

File: Resources/translations/messages.hr.xliff [hunk 1]
```diff
@@ -77,23 +77,23 @@
 			</trans-unit>
 			<trans-unit id="email.cronjob.adminNotSubmittedSubject">
 				<source>email.cronjob.adminNotSubmittedSubject</source>
-				<target state="translated">Popis neprijavljenih korisnika</target>
+				<target state="translated">[ccnet-test2] Popis neprijavljenih korisnika</target>
```
Comment (KatjaGlassConsulting) [inline Resources/translations/messages.hr.xliff:80]: please also remove [ccnet-test2] which is not necessary

Comment (milotype) [reply Resources/translations/messages.hr.xliff:80]: OK, will remove it from the croatian translation file.

**Please note:** that string is also present in the english and german translation files.
Do you want me to remove _[ccnet-test2]_ in those files as well?

Comment (milotype) [reply Resources/translations/messages.hr.xliff:80]: If you don't mind I would also update and correct the german translation file. There are some discrepancies in the wording:
- approval = Freigabe
- approved = Angenommen, Genehmigt
- approver = Genehmigender
- Timesheet/Timesheets = Stundenzettel

But I would do that in a separate/new commit/PR, after you decide upon my above comment.

Comment (KatjaGlassConsulting) [reply Resources/translations/messages.hr.xliff:80]: The current "main" branch already has the [ccnet-test2] removed. I would not like changes in the German version as this was actually used as base. I will look into the German harmonization on my own, so thanks for making me aware.

Comment (milotype) [reply Resources/translations/messages.hr.xliff]: OK, I've removed **[ccnet-test2]** from the Croatian translation.

**Pull request merged**